src/metabase/driver/sql_jdbc/connection.clj

```diff
@@ -44,7 +44,7 @@
   :hierarchy #'driver/hierarchy)
 
 (defmethod data-warehouse-connection-pool-properties :default
-  [_]
```

Comment:
I thought we should update the args here, but I wasn't aware of spec. Could you please give a bit info how it will be resolved in clojure

![image](https://user-images.githubusercontent.com/40241717/105706199-27158c00-5f09-11eb-8a26-4f32963d9600.png)

Reply:
updating args here https://github.com/revolut-engineering/metabase/pull/29/files#diff-52d394f85d35221cbf73eadbb648f468e697bea6c7c4438eed2d702740821a56R89

spec is the connection details I need to pass it the `data-warehouse-connection-pool-properties` function to conditionally assign max pool size based on `:user`

Reply:
Right, I somehow missed that one :(